### PR TITLE
feat: add extras and includeExtrasInPageSource

### PR DESCRIPTION
**Pull request by KazuCocoa**
Apply https://github.com/appium/appium-uiautomator2-server/pull/467

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +280,7 @@ useResourcesForOrientationDetection | boolean | Defines the strategy used by UiA
...
+includeExtrasInPageSource | boolean | Whether to include `extras` element attribute in the XML page source result. Then, XPath locator can find the element by the extras. The value is one string value by combining keys/values in [getExtras](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()) with `;` separator, thus you may need to find the element with partial matching like `contain` e.g. `driver.find_element :xpath, '//*[contains(@extras, "AccessibilityNodeInfo.roleDescription=")]'`. The value could be huge if elements in the XML page source had large `extras`. It could affect the XML page source generation performance.
```
Comment (jlipps) [inline README.md:283]: matching like `contains`

**Review by jlipps — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -206,6 +206,7 @@ text or name | The element's text. It never equals to null | 'my text'
 bounds | The element's visible frame (`[left, top][right, bottom]`) | `[0,0][100,100]`
 displayed | Whether the element is visible to the user | 'true'
 contentSize | The dimensions of the element's content area | `{"left": 0, "top":0, "width": 100, "height": 100, "scrollableOffset": 10, "touchPadding": 0}`
+extras | The result of [getExtras](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()). The value includes all of keys/values by combining them with `;` separator. No value for a key ends with blank. | Part of extras in chrome browser:<br> `AccessibilityNodeInfo.roleDescription=;`<br>`AccessibilityNodeInfo.chromeRole=rootWebArea;`<br> `ACTION_ARGUMENT_HTML_ELEMENT_STRING_VALUES=`<br> `ARTICLE,BLOCKQUOTE,BUTTON,CHECKBOX`
```
Comment (mykola-mokhnach) [inline README.md:209]: `No value for a key ends with blank` - could we somehow rephrase this sentence? I don't get the meaning of it

File: README.md (hunk 1 above)
Comment (mykola-mokhnach) [reply README.md:283]: Its value consists of combined  extras `keys=value` pair separated by a semicolon (`;`)

Comment (mykola-mokhnach) [inline README.md:283]: if elements in the XML page source have large

Comment (mykola-mokhnach) [reply README.md:283]: It could affect the performance of XML page source generation

File: README.md [hunk 3]
```diff
@@ -206,6 +206,7 @@ text or name | The element's text. It never equals to null | 'my text'
 bounds | The element's visible frame (`[left, top][right, bottom]`) | `[0,0][100,100]`
 displayed | Whether the element is visible to the user | 'true'
 contentSize | The dimensions of the element's content area | `{"left": 0, "top":0, "width": 100, "height": 100, "scrollableOffset": 10, "touchPadding": 0}`
+extras | The result of [getExtras](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()). The value includes all of the pair of keys and values as `key=value` separated by a semicolon (`;`). If the value is an empty, the pair will be `key=`. | Part of extras in chrome browser:<br> `AccessibilityNodeInfo.roleDescription=;`<br>`AccessibilityNodeInfo.chromeRole=rootWebArea;`<br> `ACTION_ARGUMENT_HTML_ELEMENT_STRING_VALUES=`<br> `ARTICLE,BLOCKQUOTE,BUTTON,CHECKBOX`
```
Comment (mykola-mokhnach) [inline README.md:209]: if the value is empty then only key part ending with the equal sign will be present

Comment (mykola-mokhnach) [inline README.md:209]: includes all key-value pairs

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**